arch/arm64/boot/dts/freescale/imx8mm-crocodile-no-m4.dts

```diff
@@ -5,7 +5,7 @@
 
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/interrupt-controller/irq.h>
-#include "crocodile-pinmap.h"
+#include "imx8mm-crocodile-pinmap.h"
```

Comment:
Many thanks for adapting this :+1:

We should use:
$SOC-$VENODR-$PRODUCT-$REV

so:

imx8mm-leica-corocodile-revB or
imx8mm-leicageo-crocodile-revB

Also please check the compatible. This should be:

vendor,product

compatible = "leica, crocodile-revB", "leica,crocodile", "fsl,imx8mm";

or 

compatible = "leicageo,crocodile-revB", ...

Reply:
probably s/leicageo/hexagongeo/ instead - longish ...